service/device/management/packages/job/src/main/java/org/eclipse/kapua/service/device/management/packages/job/DevicePackageDownloadTargetProcessor.java

```diff
@@ -88,4 +89,13 @@ protected JobTargetStatus getCompletedStatus(JobTarget jobTarget) {
 
         return JobTargetStatus.AWAITING_COMPLETION;
     }
+
+    @Override
+    protected String getTargetDisplayName(JobTarget jobTarget) throws KapuaException {
```

Comment:
This isn't supposed to inherit from `AbstractDeviceTargetProcessor`? 

Inheritance should be `DevicePackageDownloadTargetProcessor` > `AbstractDevicePackageTargetProcessor` > `AbstractDeviceTargetProcessor` > `AbstractTargetProcessor`> etc